index.html

```diff
@@ -5,8 +5,55 @@
     <meta http-equiv="X-UA-Compatible" content="IE=edge">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Weather Report</title>
+    <link href="./styles/index.css" rel="stylesheet">
 </head>
 <body>
-    
+    <header>
+        <h1>WEATHER REPORT</h1>
+    </header>
+    <!-- <section> -->
+        <div class="left-column box" id="temp-input">
+            <h2 id="temperature-text">TEMPERATURE</h2>
+
+            <button type="button" id="increase-temp">⬆️</button>
+            <p id="temp">72</p>
+            <button type="button" id="decrease-temp">⬇️</button>
+            <button type="button" id="realtime-temp">Get Realtime Temperature</button>
+        </div>
+        <div class="left-column box" id="sky-input">
+            <h2 id="sky-text">SKY</h2>
+            <!-- dropdown source: https://www.w3schools.com/howto/howto_js_cascading_dropdown.asp -->
+            <form id="dropdown"> <!-- Not sure what this is supposed to look like -->
+                <select name="sky" id="sky-select">
+                <option value="sunny">Sunny</option>
+                <option value="cloudy">Cloudy</option>
+                <option value="rainy">Rainy</option>
+                <option value="snowy">Snowy</option>
+                </select>
+                <br><br>
+            </form>
+        </div>
+        <div class="left-column box" id="city-input">
+            <h2>CITY NAME</h2>
+            <!-- blank field source: https://www.w3schools.com/tags/tryit.asp?filename=tryhtml_form_target-->
+            <form action="/action_page.php" method="get" target="_blank">
+                <input type="text" id="city-name-input" name="city-name-input">
+            </form>
```

Comment:
I there are better ways to do this using JavaScript. Right now your form attempts to use `action_page.php` - something that doesn't exist on your site. It's OK to look up how to do things online, but tey to always make sure you understand what each part is doing.